storagenode/storagenodedb/storagenodedbtest/satellites_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedbtest_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode"
+	"storj.io/storj/storagenode/satellites"
+	"storj.io/storj/storagenode/storagenodedb/storagenodedbtest"
+)
+
+// TestGracefulExitDB tests the graceful exit database calls
+func TestGracefulExitDB(t *testing.T) { //satelliteID storj.NodeID, finishedAt time.Time, exitStatus satelliteStatus, completionReceipt []byte) (err error) {
```

Comment:
Let's test with 2 distinct satellite Node IDs. The test looks good overall besides that.

Reply:
done